### PR TITLE
sti create fixes

### PR DESCRIPTION
**Pull request by soltysh**
Update sti create command with:
 - file permissions of created files
 - contents of all created files

@mfojtik @bparees ptal

### CODE REVIEW

**Comment by mfojtik** [comment]
@soltysh can you also provide template for the openshift labels in docker template?

**Comment by soltysh** [comment]
@mfojtik done

File: pkg/create/templates/docker.go [hunk 1]
```diff
@@ -4,16 +4,33 @@ const Dockerfile = `
...
+# RUN yum install -y ... && yum clean all -y
 
-USER default
+# This default user is created in the openshift/base-centos7 image
+USER 1001
 
 # TODO (optional): Copy the builder files into /opt/openshift
 # COPY ./<builder_folder>/ /opt/openshift/
 
+# TODO: Copy the STI scripts /usr/local/sti, since openshift/base-centos7 image sets io.s2i.scripts-url label that way, or update that label
```
Comment (bparees) [inline pkg/create/templates/docker.go]: Copy the STI scripts *to* /usr/local/sti

**Comment by bparees** [comment]
one nit and lgtm.

**Comment by soltysh** [comment]
@bparees fixed.

**Comment by bparees** [comment]
[merge]

**Comment by soltysh** [comment]
I left there some nasty error, failing the compilation :disappointed:. Fixed now.

**Pull request merged**